onnxscript/onnx_types.py

```diff
@@ -2,20 +2,17 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 # --------------------------------------------------------------------------
+
 from __future__ import annotations
 
-from typing import Optional, Tuple, Union
+from abc import ABC
```

Comment:
I would import abc and use abc.ABC

Reply:
Any particular reason?

Reply:
Mostly because of this https://github.com/microsoft/onnx-script/wiki/Coding-style#avoid-relative-imports-import-only-modules, as suggested by https://google.github.io/styleguide/pyguide.html#23-packages.

`ABC` is usually unique enough, but I think maintaining a single rule of importing only modules can save us from deciding what to import as modules and what not to.